src/dodal/cli.py

```diff
@@ -39,9 +45,20 @@ def main(ctx: click.Context) -> None:
     "attempt any I/O. Useful as a a dry-run.",
     default=False,
 )
-def connect(beamline: str, all: bool, sim_backend: bool) -> None:
+@click.option(
```

Comment:
I think this file can all be reverted in favour of just connecting to whatever $BEAMLINE is.

Reply:
alreadu discussed above